library/oneview_scope.py

```diff
@@ -14,16 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###
```

Comment:
The `# -*- coding: utf-8 -*-` wasn't included in this file.
Also missing update the (2016-2017)